app/src/main/res/values/styles.xml

```diff
@@ -15,8 +15,54 @@
         <item name="android:backgroundTint">@color/button_main_activity_cream</item>
     </style>
 
+    <style name="InputStyleEntryDatePickerReservationActivity">
+        <item name="android:inputType">date</item>
+        <item name="android:focusable">false</item>
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_entry_date_margin_top</item>
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_entry_date_text_size</item>
+        <item name="android:clickable">true</item>
+        <item name="android:maxLines">1</item>
+    </style>
+
+    <style name="InputStyleExitDatePickerReservationActivity">
+        <item name="android:inputType">date</item>
+        <item name="android:focusable">false</item>
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_entry_date_margin_top</item>
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_entry_date_text_size</item>
+        <item name="android:clickable">true</item>
+        <item name="android:maxLines">1</item>
+    </style>
+
+    <style name="InputStyleReservationActivityParkingNumber">
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_parking_number_margin_top</item>
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_parking_number_text_size</item>
+    </style>
+
+    <style name="InputStyleReservationActivityKeyCode">
+        <item name="android:textSize">@dimen/edit_text_reservation_activity_parking_code_text_size</item>
+        <item name="android:layout_marginTop">@dimen/edit_text_reservation_activity_parking_code_margin_top</item>
+    </style>
+
+    <style name="ButtonStyleReservationActivitySave">
```

Comment:
a style with only one property it not make sense, please remove this style and put the property in the XML view or add more properties here :P

Reply:
done